hbase-server/src/test/java/org/apache/hadoop/hbase/master/normalizer/TestSimpleRegionNormalizer.java

```diff
@@ -370,6 +371,35 @@ public void testHonorsMergeEnabledInTD() {
 
   @Test
   public void testHonorsMinimumRegionCount() {
+    conf.setInt(MERGE_MIN_REGION_COUNT_KEY, 1);
```

Comment:
Can you instead keep a single implementation in a helper method that allows for the configuration used to be passed in as a parameter? This would be better than copy-pasting the unit test method body.